doc/ssh_config/ssh_config.md

```diff
@@ -161,6 +175,20 @@ module sonic-ssh-server {
                         }
                     }
 				}
+				leaf auto_logout {
+					description "inactivity timeoout (min unit)";
+					default 15;
+					type uint32 {
+						range 0..35000;
+					}

```

Comment:
Please add description for 0 value to indicate no auto logout.

Reply:
Done.